ts/modules/item-series.src.ts

```diff
@@ -199,6 +199,9 @@ seriesType<Highcharts.ItemSeries>(
     {
         markerAttribs: void 0,
         translate: function (this: Highcharts.ItemSeries): void {
+            if (this.total === 0) {
```

Comment:
Add a short comment on why we are doing this, and a reference to the issue number.